docs/pages/inboxes/build-inbox.mdx

```diff
@@ -922,7 +937,7 @@ await alix.conversations.findConversationByTopic(conversation.topic);
 // Returns a Group
 await alix.conversations.findGroup(group.id);
 // Returns a DM
-await alix.conversations.findDmByAddress(bo.address);
+await alix.conversations.findDmByInboxId(bo.inboxId);
```

Comment:
```suggestion
await alix.conversations.findDmByIdentity(bo.identity);
```

Reply:
We already have a method for finding by inboxId we expose one for identity now too.